### PR TITLE
APP-2029 - Add GetAuthorizationsForUserInOrg for permissions lookups

### PR DESCRIPTION
**Pull request by acrantel**
Needed for the detailed permissions view on the org settings page.
Pulls all the authorizations in the organization for a user

This is the first time I've written a protobuf so would appreciate comments on naming / other conventions.

https://github.com/viamrobotics/app/pull/1941

### CODE REVIEW

File: proto/viam/app/v1/app.proto [hunk 1]
```diff
@@ -723,6 +725,16 @@ message ListAuthorizationsResponse {
   repeated Authorizations authorizations = 1;
 }
 
+message GetAuthorizationsForUserInOrgRequest {
+  string identity_id = 1;
+  string organization_id = 2;
+}
+
+message GetAuthorizationsForUserInOrgResponse {
+  string identity_id = 1;
```
Comment (Kschappacher) [inline proto/viam/app/v1/app.proto:734]: You can remove this. We should already have the identity_id from when send the GetAuthorizationsForUserInOrgRequest

**Review by Kschappacher — APPROVED** [review]
Names look good, just one nit

**Comment by acrantel** [comment]
do not merge yet, identity authorizations are no longer 1:1 to resources. need further investigation before adding this endpoint

**Comment by acrantel** [comment]
superseded by https://github.com/viamrobotics/api/pull/284

**Pull request closed without merging**